stable/swagger/values.yaml

```diff
@@ -0,0 +1,63 @@
+replicaCount: 1
+
+env:
+  # API_URL is a location of swagger.json file
+  # e.g.:
+  #   http://generator.swagger.io/api/swagger.json
+  API_URL:
+
+image:
+  repository: swaggerapi/swagger-ui
+  tag: v3.14.1
+  pullPolicy: IfNotPresent
+
+service:
+  portName: http
+  protocol: TCP
+  type: NodePort
+  externalPort: 80
+
+ingress:
+  enabled: false
+  # e.g.:
+  # - swagger.example.com
+  hostname:
+
+  ##  When using nginx-ingress & cert-manager, set these annotations,
+  ##      and use the below tls setup
+  annotations: {}
+  #   kubernetes.io/ingress.class: nginx
+  #   kubernetes.io/tls-acme: 'true'
+  tls: []
+  # - hosts:
+  #   - swagger.example.com
+  #   secretName: swagger-tls
+
```

Comment:
Basic auth an OAuth stuff is specific to nginx-ingress. Make annotations generally configurable instead.